### ISSUE TITLE
Mark Defeated not rolled back

### ISSUE THREAD

**Issue by HippyCraig** [death-save/combat-utility-belt#465]
Not sure if this is a bug or by design, but I created a new condition that will mark an NPC defeated but if there is remove option as I will show in the screen shot that removes the condition but doesn't reverse the Mark Defeated.

![image](https://user-images.githubusercontent.com/17074456/104960896-3b0a3c80-59a3-11eb-9138-a9d1e46aeb1f.png)

 - Foundry Version: [e.g. 0.7.9]
 - Game System / Version: [e.g. dnd5e 1.2.2]
 - CUB Version [e.g. 1.3.8]
 - Browser [chrome]
 - OS [e.g. Windows 10]

**Comment by eclarke12** [comment]
@HippyCraig you're right there's no explicit behaviour to "undefeat" a combatant currently.

**Comment by HippyCraig** [comment]
is that something that will be added?  If not I have been trying to write a macro but cant seem to get it to work.

**Comment by eclarke12** [comment]
> is that something that will be added? If not I have been trying to write a macro but cant seem to get it to work.

Triggler doesn't currently pass the affected entity to the macro, so it's not really feasible to macro.

I will use this issue as a reminder to add this functionality to CUB

**Comment by HippyCraig** [comment]
Any idea when this will make it in?
